index.html

```diff
@@ -0,0 +1,273 @@
+            <img src="image/Snapshoot_Portfolio(3).svg" class="img" alt="stories">
+          </div>
+        <div>
+          <h4>Multi-Post Stories</h4>
+          <div class="menu">
+            <ul class="Tonic-menu">
+              <li>CANOPY</li>
+              <li>Back End Dev</li>
+              <li>2015</li>
+            </ul>
+          </div>
+          <p class="instruction">A daily selection of privately personalized reads;
+            no accounts or sign-ups required.
+          </p>
+          <div class="language">
+            <ul>
+              <li>html</li>
+              <li>css</li>
+              <li>javaScript</li>
+            </ul>
+          </div>
+          <div class="btn-T">
+            <button type="button">See Project</button>
+          </div>
+          </div>
+      </li>
+      <li class="box">
+        <div class="Tonic">
+          <img src="image/Snapshoot_Portfolio(2).svg" class="img" alt="facebook">
+        </div>
+      <div>
+        <h4>Tonic</h4>
+        <div class="menu">
+          <ul class="Tonic-menu">
+            <li>CANOPY</li>
+            <li>Back End Dev</li>
+            <li>2015</li>
+          </ul>
+        </div>
+        <p class="instruction">A daily selection of privately personalized reads;
+          no accounts or sign-ups required.
+        </p>
+        <div class="language">
+          <ul>
+            <li>html</li>
+            <li>css</li>
+            <li>javaScript</li>
+          </ul>
+        </div>
+        <div class="btn-T">
+          <button type="button">See Project</button>
+        </div>
+      </div>
+    </li>
+    <li class="box">
+        <div class="Tonic">
+          <img src="image/SnapshootPortfolio(1).svg" class="img" alt="available">
+        </div>
+      <div>
+        <h4>Multi-Post Stories</h4>
+        <div class="menu">
+          <ul class="Tonic-menu">
+            <li>CANOPY</li>
+            <li>Back End Dev</li>
+            <li>2015</li>
+          </ul>
+        </div>
+        <p class="instruction">A daily selection of privately personalized reads;
+          no accounts or sign-ups required.
+        </p>
+        <div class="language">
+          <ul>
+            <li>html</li>
+            <li>css</li>
+            <li>javaScript</li>
+          </ul>
+        </div>
+        <div class="btn-T">
+          <button type="button">See Project</button>
+        </div>
+      </div>
+    </li>
+  </ul>
+  </section>
+  <section class="aboutMe">
+    <div class="headline-main-container">
+      <h3 class="welcome">About <br> Myself </h3> 
+      <p class="introduction">
+        Hello I’m a software developer! I can help you build a product ,
+        feature or website Look through some of my work and experience! 
+        If you like what you see and have a project you need coded,
+        don’t hestiate to contact me. 
+      </p>
+      <div class="link-up">
+        <h2>LET'S CONNECT</h2>
+        <div class="social-list"></div>
```

Comment:
Add some space between the text and the icons
### Original design
![right](https://user-images.githubusercontent.com/58376020/126038902-23c9c3c4-aa4e-4be4-abbc-bd43d7716fb9.png)
### Your Project
![wrong](https://user-images.githubusercontent.com/58376020/126038910-6379b397-1323-4a06-aefa-5e97b5ac909c.png)